test_esss_jenkins.py

```diff
@@ -229,6 +229,17 @@ def test_bhist(jenkins_plugin, testbot, mocker, LineMatcher):
     assert settings['last_job_listing'] == ['job-A', 'job-B']
 
 
+def test_fetch_job_status(jenkins_plugin, mocker):
+    mocker.patch.object(jenkins_plugin,
+                        '_get_jenkins_json_request',
+                        return_value={"_class": "hudson.model.FreeStyleBuild", "result": None})
+    jenkins_plugin._fetch_job_status("dummy")
+    mocker.patch.object(jenkins_plugin,
+                        '_get_jenkins_json_request',
+                        return_value={"_class": "hudson.model.FreeStyleBuild"})
```

Comment:
Was this the returned value triggering the error?

Reply:
Yep